src/components/ListingCard.svelte

```diff
@@ -26,36 +26,60 @@
       updatedListingLikes = [item, ...updatedListingLikes];
     });
     listingsWithLikes = updatedListingLikes;
+    console.log(listingsWithLikes);
```

Comment:
Console log left in, not sure if it's needed.

Reply:
Removed